modin/experimental/core/execution/native/implementations/hdk_on_native/dataframe/dataframe.py

```diff
@@ -1965,10 +1963,9 @@ def _materialize(self):
 
         if self._force_execution_mode != "hdk" and self._can_execute_arrow():
             new_table = self._execute_arrow()
-            partitions = np.empty((1, 1), dtype=np.dtype(object))
-            partitions[0][0] = self._partition_mgr_cls._partition_class.put_arrow(
-                new_table
-            )
+            partitions = self._partition_mgr_cls.from_arrow(
+                new_table, unsupported_cols=False, encode_col_names=False
+            )[0]
```

Comment:
?
```suggestion
            partitions, _ = self._partition_mgr_cls.from_arrow(
                new_table, unsupported_cols=False, encode_col_names=False
            )
```

Reply:
I think, the first variant is a bit more efficient, because it does not unpack the tuple, and does not put the value into the unused variable.